frontend/src/components/NewTopicsEditor.tsx

```diff
@@ -0,0 +1,333 @@
+import { useMutation } from '@apollo/client';
```

Comment:
This file is pretty long, which makes it a bit hard to find what you're looking for in it. I see that multiple components are written here - let's split them to separate files in `components/NewTopicsEditor/` and also create an `index.ts` file in that dir, that would export all of the components.

Reply:
@avirankatz Since it's not blocking functionality or operability, can this refactor be part of a follow-up task?